SPECS/bluez/bluez.spec

```diff
@@ -2,7 +2,7 @@ Vendor:         Microsoft Corporation
 Distribution:   Mariner
 Name:    bluez
 Version: 5.63
-Release: 1%{?dist}
+Release: 2%{?dist}
```

Comment:
Release bumps should be accompanied with an entry in the `%changelog` section

Reply:
changelog updated